packages/frontend/core/src/components/explorer/context.ts

```
@@ -49,6 +49,9 @@ export const createDocExplorerContext = (
     selectedDocIds$: new LiveData<string[]>([]),
     prevCheckAnchorId$: new LiveData<string | null>(null),
     displayPreference$: displayPreference$,
+    showDragHandle$: displayPreference$.selector(
+      displayPreference => displayPreference.showDragHandle
+    ),
```

Biome (1.9.4)

[error] This property value named showDragHandle$ is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named showDragHandle$


(lint/suspicious/noDuplicateObjectKeys)